lib/twiml_generator.rb

```diff
@@ -1,18 +1,21 @@
 module TwimlGenerator
-  def self.generate_connect_conference(conference_id, wait_url, start_on_enter, end_on_exit)
-    Twilio::TwiML::Response.new do |r|
-      r.Dial do |d|
-        d.Conference conference_id, startConferenceOnEnter: start_on_enter,
-                               endConferenceOnExit: end_on_exit,
-                               waitUrl: wait_url
-      end
-    end.to_xml
+  def self.generate_connect_conference(conference_id, wait_url,
+                                       start_on_enter, end_on_exit)
+    r = Twilio::TwiML::VoiceResponse.new
+    d = Twilio::TwiML::Dial.new
+    d.conference(conference_id, startConferenceOnEnter: start_on_enter,
```

Comment:
Named attributes uses underscore, not camelCase : https://github.com/twilio/twilio-ruby/blob/next-gen/lib/twilio-ruby/twiml/voice_response.rb#L376